tests/variables.tf

```diff
@@ -0,0 +1,7 @@
+variable "codebuild_id" {
```

Comment:
let's change this to `build_id`. no reason to tie it to codebuild, specifically.

Reply:
changed